### PR TITLE
Fixed a refresh bug on the bme as well as ID issues

### PR DESCRIPTION
**Pull request by Tysonpower**
I got a customer that had issues with the updateing rate from the webui ( default 120s) not working.

The BME sensor was only updating every 2h (forced update in code), the bug was a missing "New_data" flag in the "if USE_BME" statement, fixed that and changed a i2c id so it works with the bme sensor, maybe we should add a webui field or the i2c id in the future.

All was tested on my own unit with a bme280 sensor, all works fine.

regards,
Manuel

### CODE REVIEW

**Comment by Gifford47** [comment]
the changes are not valid. there are different adresses for bme (0x76 and 0x77). 0x76 is the default. therefore is should be also the default in the code.
the ```new_sensor_data = true;``` forces an mqtt update of the sensors every time after reading. it bypasses the thresholds.

i guess the fault is somewhere else...

File: HCPBridgeESP32/src/main.cpp [hunk 1]
```diff
@@ -771,6 +772,7 @@ void SensorCheck(void *parameter){
         bme_temp = bme.readTemperature();   // round float
         bme_hum = bme.readHumidity();
         bme_pres = bme.readPressure()/100;  // convert from pascal to mbar
+        new_sensor_data = true;
```
Comment (Gifford47) [inline HCPBridgeESP32/src/main.cpp:775]: bypasses the thresholds...

**Comment by Tysonpower** [comment]
@Gifford47 found the real issue now, the prox threshold was used for all sensor, also added a small info text to the ui for the thresholds.

File: HCPBridgeESP32/src/main.cpp [hunk 2]
```diff
@@ -303,6 +303,7 @@ void updateDoorStatus(bool forceUpate = false)
 }
 
 void updateSensors(bool forceUpate = false){
+  Serial.println("Updating sensors");
```
Comment (Gifford47) [inline HCPBridgeESP32/src/main.cpp:306]: please remove it or use it with DEBUG define. it´s only useful for debugging

File: HCPBridgeESP32/src/main.cpp [hunk 3]
```diff
@@ -771,6 +772,7 @@ void SensorCheck(void *parameter){
         bme_temp = bme.readTemperature();   // round float
         bme_hum = bme.readHumidity();
         bme_pres = bme.readPressure()/100;  // convert from pascal to mbar
+        //new_sensor_data = true;
```
Comment (Gifford47) [inline HCPBridgeESP32/src/main.cpp:775]: pls, remove

File: HCPBridgeESP32/src/main.cpp [hunk 4]
```diff
@@ -1010,7 +1012,7 @@ void setup()
       i2c_sclpin = localPrefs->getInt(preference_sensor_i2c_scl);
       pinMode(i2c_onoffpin, OUTPUT);
       I2CBME.begin(i2c_sdapin, i2c_sclpin);   // https://randomnerdtutorials.com/esp32-i2c-communication-arduino-ide/
-      bme_status = bme.begin(0x76, &I2CBME);  // check sensor. adreess can be 0x76 or 0x77
```
Comment (Gifford47) [inline HCPBridgeESP32/src/main.cpp:1013]: [default address](https://learn.adafruit.com/adafruit-bme280-humidity-barometric-pressure-temperature-sensor-breakout/pinouts#i2c-address-3153810) should be 0x76, as most ic`s use it.

**Comment by Gifford47** [comment]
please open a new PR with the above changes. i`ll close this now

**Pull request closed without merging**